docker-compose.yml

```diff
@@ -12,6 +12,8 @@ services:
   db:
     image: postgres:11.5-alpine
     ports:
-      - "5432:5432"
+      - "5433:5432"
```

Comment:
I'm not sure about that. I use `compose` only for local development, and the default port is almost perfect for me. I agree that the port may already be used on the local machine, but in this case it might be better to disable port mapping completely?

Reply:
Thanks for the quick reply! I wouldn't recommend disabling it at least in development environments because usually there is a need to connect to DB (running in a container) on the host machine. That's why we do port mapping. The mapping "5432:5432" will cause an issue if you run something on port 5432 on your host machine (usually Postgres). Also take a note that Docker writes directly to ip table, so firewalls won't stop docker to expose ports. It's worth to read: https://stackoverflow.com/questions/30383845/what-is-the-best-practice-of-docker-ufw-under-ubuntu and https://docs.docker.com/network/iptables/. In my production environment `127.0.0.1:5432:5432` works well.

Reply:
> The mapping "5432:5432" will cause an issue if you run something on port 5432 on your host machine (usually Postgres).

But docker will print a nice error that port is already is use(okay, not quite nice, but understandable):
```bash
ERROR: for fastapi-realworld-example-app_db_1  Cannot start service db: driver failed programming external connectivity on endpoint fastapi-realworld-example-app_db_1 (6fd471df68f2da20a2bd9a711a5864b3ff87b09a8379f4d80f505a8eedcc0d43): Bind for 0.0.0.0:5432 failed: port is already allocated
```

Then the user can decide for himself whether he wants to create a new instance of postgres on another port(and manually change it in `docker-compose.yml`) or use an existing one, right? I don’t have postgres on my laptop, and even if it was there, changing the port to 5433 wouldn’t help if some other project also changed the port for the database, right? Isn’t it better to leave the defaults, allowing the user to configure for himself if necessary?

> Also take a note that Docker writes directly to ip table, so firewalls won't stop docker to expose ports.

Yes, I agree. For this reason I prefer to deploy all containers on my servers via `traefik` and export only its ports (80 and 443 usually).

Reply:
+ Not need to check which port is opened, you that this will be default one
...

- The annoying message that port is busy, either stop local service or change it
- If you changed it, it's easy to forget and commit. For me, it happens quite often
...

So, we need to decide)

Reply:
As an additional argument for annoying message:

I usually use `pgcli` to work with postgres(although this also applies to `psql`). And it seems to me that it will bother to always use the custom port for connecting to the project database, dont you think so?

Reply:
Nice! Then it remains to add the folder for docker and fix Github Actions CI. If I get it right, then the files `conduit.yml`, `styles.yml` and `tests.yml` should be a little changed to this to start CI for this PR:
```yaml
on: [push, pull_request]
```

And maybe issue #40 can also be closed in this PR? What do you think about this?

Reply:
@nsidnev Cool, I will also try to add this small change.